src/libraries/Microsoft.Extensions.Logging/tests/Common/ConsoleLoggerTest.cs

```diff
@@ -1170,6 +1176,82 @@ public void ConsoleLoggerOptions_IncludeScopes_IsReadFromLoggingConfiguration()
             Assert.NotNull(logger.ScopeProvider);
         }
 
+        [Fact]
+        public void TestActivityIds()
+        {
+            RemoteExecutor.Invoke(() =>
```

Comment:
Why do you need this?

Reply:
This to run the test in its own process so I can capture all console output isolated from any other tests.

Reply:
Test it without the console logger. It would be cleaner. All we need to verify is that the scope information contains the right activity fields.

Reply:
ok

Reply:
@davidfowl I have updated the test as you have suggested. Please let me know if you have any other comments. Thanks!